design/sketch.webidl

```diff
@@ -428,14 +428,46 @@ interface WebGPURenderPipeline {
 // COMMAND RECORDING (Command buffer and all relevant structures)
 // ****************************************************************************
 
-enum WebGPULoadOp {
+/// Common interface for render and compute pass encoders.
+interface WebGPUCommandEncoder {
+    WebGPUCommandBuffer end_pass();
```

Comment:
Is this intended to always return the command buffer that the pass was created inside of? Seems somewhat confusing to have this return anything at all.